app/models/condition.rb

```diff
@@ -189,8 +192,9 @@ def self._subst_find(rec, inputs, expr)
 
     if checkmode == "count"
       e = check.gsub(/<count>/i, list.length.to_s)
+      left, operator, right = e.split
       MiqPolicy.logger.debug("MIQ(condition-_subst_find): Check Expression after substitution: [#{e}]")
-      result = !!proc { $SAFE = 3; eval(e) }.call
+      result = !!left.to_f.send(operator, right.to_f)
```

Comment:
If we can't remove the send, can we whitelist the operator in a constant containing `%w(=, !=, <, > <=, >=)` and raise an error if the operator isn't in the whitelist?  Maybe also a public_send?  If we can't remove the send, we should ensure it's only allow types of operators that are called.